contrib/builder/deb/aarch64/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+cd "$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
I don't think these are the same since readlink -f will return the full path, but in my quick test BASH_SOURCE does not.